runtime/doc/filetype.txt

```diff
@@ -544,6 +544,8 @@ CTRL-T                    Jump back to the location that the manpage was
                           opened from.
 q                         :quit if invoked as $MANPAGER, otherwise :close.
 
+To see the table of contents, type <M-]>.
```

Comment:
Which is more accurate/helpful to use in the docs?  `META-` or `ALT-`?  I feel like `ALT-` should be used because it's actually on the keyboard.  But, I don't know enough to know if `META-` is always the same as `ALT-`.  My vague recollection is that they aren't always the same.